_docs-sources/pipelines/pipelines-actions/index.md

```diff
@@ -0,0 +1,21 @@
+# Pipelines Actions
+
+When a user opens a Pull Request, Pipelines runs some set of "actions" in response to the proposed [infrastracture changes](../core-concepts/#infrastructure-change). We call these _pipelines actions_. Gruntwork Pipelines supports the following pipelines actions:
+
+## Terragrunt plan
+
+When the pull request is created, Pipelines will automatically execute `terragrunt plan` on every infrastructure-change. The output will be written to a Pull Request comment.
+
+## Terragrunt apply/destroy
+
+When the pull request is merged, Pipelines will automatically execute either `terragrunt apply` or `terragrunt destroy` on every infrastructure change, depending on the type of infrastructure change. For example, if the pull request deletes a `terragrunt.hcl` file, Pipelines will run `terragrunt destroy`.
+
+## Coming soon
```

Comment:
I'd remove this section entirely or move its content to the marketing site

Reply:
+1. We shouldn't document behavior that doesn't exist yet.

Reply:
I wanted to convey the concept of Pipelines Actions, but your (plural) instincts to keep all future promises out of docs is probably the right call.